connector_search_engine/models/se_binding.py

```diff
@@ -60,6 +66,9 @@ def recompute_json(self, force_export=False):
             mapper = work.component(usage='se.export.mapper')
             lang = work.index.lang_id.code
             for record in work.records.with_context(lang=lang):
+                # TODO: check WTH here it was converted to list
```

Comment:
It was not converted in list orignaly, the "list" method have been introduced by the commit that convert to python 3 https://github.com/akretion/connector-search-engine/pull/41/commits/7d8402aaa18e8d991388e39624feaddc251f62ba#diff-9784769f86a072668113a66d47d6b5f2R62

Reply:
ok, then we can drop the comment :)